src/client/setup.ts

```diff
@@ -46,7 +48,13 @@ function twitter(): void {
 function performanceMetrics(): void {
     window.addEventListener(
         'load',
-        () => console.log('Metrics: ', metrics(performance.getEntries())),
+        () => {
+            pipe2(
+                performance.getEntries(),
+                metrics,
+                metricsClient.sendMetrics.bind(null),
```

Comment:
why do you need to bind here?

Reply:
ESLint complains about using unbound methods otherwise, due to [this rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unbound-method.md). I believe the same warning prompted [this](https://github.com/guardian/apps-rendering/blob/27f8f65a54a36fa452dbc183761e7e0755978773/src/server/server.ts#L54).